packages/lesswrong/components/common/Header.tsx

```diff
@@ -154,6 +165,208 @@ const styles = (theme: ThemeType): JssStyles => ({
       position: "fixed !important",
     },
   },
+  
+  // Below are custom styles for the special EA Forum Giving Season 2023 homepage header
+  // TODO: delete after 2023
+  rootGivingSeason: {
+    height: EA_FORUM_GIVING_SEASON_HEADER_HEIGHT,
+    [theme.breakpoints.down('sm')]: {
+      height: EA_FORUM_HEADER_HEIGHT,
+    },
+    "& .headroom": {
+      zIndex: theme.zIndexes.searchResults,
+    },
+  },
+  appBarGivingSeason: {
+    color: theme.palette.givingPortal.homepageHeader.light4,
+    background: `center no-repeat url(${gsImg}), ${theme.palette.givingPortal.homepageHeader.dark}`,
```

Comment:
This centering is visually off from the center column for me on my very wide screen. May or may not be worth caring about.

Reply:
Screenshot? Also the center column can change position based on whether or not the RHS is visible, and I don't think the header image should move if the center column moves, so I'm just going to leave it as-is (which is actually in the center)

Reply:
<img width="2992" alt="image" src="https://github.com/ForumMagnum/ForumMagnum/assets/10352319/5fba54e9-b688-41b1-be22-1542e69b65bc">